testing/nginx-naxsi/nginx.initd

```diff
@@ -1,67 +1,40 @@
 #!/sbin/openrc-run
+supervisor=supervise-daemon
 
 description="Nginx http and reverse proxy server"
 extra_started_commands="reload reopen upgrade"
 
 cfgfile=${cfgfile:-/etc/nginx/nginx.conf}
-pidfile=/run/nginx/nginx.pid
+pidfile=/run/nginx/$SVCNAME.sd.pid
```

Comment:
This variable should be named `$RC_SVCNAME`, please read [openrc-run(8)](http://manpages.org/openrc-run/8#environment).